open_event/static/admin/js/map.js

```diff
@@ -1,13 +1,13 @@
 $( document ).ready(function() {
 
-    var long = $('#longitude').val();
-    var lati = $('#latitude').val();
-    var label_name = $('#name').val();
-    if (lati == '') {
-        lati = '51.08150695757747'
+    var long = $("#longitude").val();
+    var lati = $("#latitude").val();
+    var labelName = $("#name").val();
```

Comment:
I don't really understand this issue. I think var is the right thing to be used here.